src/screens/landingPage/components/whoAreWe/components/collaboratorsList/index.tsx

```diff
@@ -0,0 +1,50 @@
+import Carousel from 'react-material-ui-carousel';
+import theme from '../../../../../../utils/theme';
+import { collaborators } from '../../../../constants';
+import CollaboratorsListItem from '../collaboratorsListItem';
+import { CollaboratorsContainer, Container } from './styles';
+
+const CollaboratorsList = () => {
+  return (
+    <>
+      <Container>
+        <Carousel
+          animation="slide"
+          duration={1000}
+          interval={8000}
+          navButtonsAlwaysInvisible={true}
+          activeIndicatorIconButtonProps={{
+            style: {
+              color: theme.palette.primary.light,
+            },
+          }}
+        >
+          <CollaboratorsContainer>
+            {collaborators[0].map((collaborator) => (
+              <CollaboratorsListItem
+                key={collaborator.name}
+                linkedin={collaborator.linkedin}
+                name={collaborator.name}
+                role={collaborator.role}
+                photo={collaborator.photo}
+              />
+            ))}
+          </CollaboratorsContainer>
+          <CollaboratorsContainer>
+            {collaborators[1].map((collaborator) => (
+              <CollaboratorsListItem
+                key={collaborator.name}
+                linkedin={collaborator.linkedin}
+                name={collaborator.name}
+                role={collaborator.role}
+                photo={collaborator.photo}
+              />
+            ))}
+          </CollaboratorsContainer>
```

Comment:
Ao invés de acessar diretamente os índices 0 e 1 o melhor seria fazer um map na lista. Algo mais ou menos assim:
```suggestion
          {
            collaborators.map((collabs) => (
              <CollaboratorsContainer>
                {collabs.map((collaborator) => (
                  <CollaboratorsListItem
                    key={collaborator.name}
                    linkedin={collaborator.linkedin}
                    name={collaborator.name}
                    role={collaborator.role}
                    photo={collaborator.photo}
                  />
                ))}
              </CollaboratorsContainer>
            ))
          }
```

Reply:
feito